README.md

```diff
@@ -69,116 +70,115 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
+- Added ARM64 configurations to the remaining projects.
+- Fixed PowerRename to work on ARM64.
+- Fixed File Explorer tools to work on ARM64.
+- Made changes for the installer projects to build ARM64 installers.
+- Configured the CI and Release pipelines to build for ARM64.
+- Added ARM64 build status to the README.
 
 ### Always on Top
 
-- Fix for topmost state of the window resetting for some applications. (This was a hotfix for 0.57)
+- Fixed an issue where the borders where sticking around when a window was minimized with Win+D.
 
-### ColorPicker
+### FancyZones
 
-- The CIEXYZ format is now properly show in upper case.
+- Fixed a bug that was consuming CPU cycles when the default layout was set.
+- Fixed a bug where apps were not opened in their last known zones due to Virtual Desktop ID changes.
+- Fixed a bug that was snapping popup menus opened by applications.
+- Fixed a bug causing windows not to be snapped under some configurations.
 
-### FancyZones
+### Image Resizer
 
-- Restore rounded corners on Windows 11 and add a setting to control this behavior. (This was a hotfix for 0.57)
-- Fixed an edge case where the Windows Terminal window wouldn't be snapped when opened. (This was a hotfix for 0.57)
-- Improved narrator support in the Grid Editor. (This was a hotfix for 0.57)
-- Fixed a bug when restoring rounded corners on Windows 11. (This was a hotfix for 0.57)
-- Fix for windows not being resized correctly on different dpi settings. (This was a hotfix for 0.57)
-- Removed resolution from the screen identifier so zones aren't reset when resolution changes.
-- Scale the canvas layout when editing according to new scaling/resolution.
-- Shipping a new tool to help debug windows interactions with FancyZones.
+- No longer tries to change metadata on files that were not actually resized. Thanks [@adamchilders](https://github.com/adamchilders)!
 
-### File explorer
+### File explorer add-ons
 
-- Fix for a crash in dev file preview if the settings file hadn't been created yet. (This was a hotfix for 0.57)
-- New file types were added to dev file preview (".reg", ".xslt", ".xsd", ".wsdl", ".ino", ".pde", ".razor"). Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
-- Fix an existing "file still in use" issue in dev file preview. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
-- Dev file preview is now able to interpret file extensions in a case-insensitive way. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
-- SVG and markdown viewers no longer use WebBrowser and use WebView2 instead.
-- Markdown preview now respects the dark mode settings on Windows. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+- Fixed a bug where modules depending on WebView2 would be limited to opening files smaller than 2 MB. Now the resulting html is generated into a temporary file before presenting it.
+- Add a viewBox attribute to svgs that don't have one so that the preview tries to show the whole image.
```

Comment:
```suggestion
- Add a viewBox attribute to svg files that don't have one so that the preview tries to show the whole image.
```
(it's a nit, i know)